backend/src/backend/exporters/__init__.py

```diff
@@ -0,0 +1,20 @@
+import pathlib
+import tempfile
+
+import openzim_uploader
+
+from backend.constants import BackendConf
+
+
+async def exporter(data):
```

Comment:
The naming is also incorrect. `exporter` doesn't mean anything. This is a an `OpenzimUploadExporter` and it should be in a dedicated module.

here you should just have the empty interface